### PR TITLE
Burnin container

### PR DESCRIPTION
**Pull request by ctierneynv**
Fixed pull request to reorganize and update BIT to use NGC container

### CODE REVIEW

**Comment by avolkov1** [comment]
I am reviewing this now.

**Review by avolkov1 — CHANGES_REQUESTED** [review]
There was one typo for command when running with singularity. Please fix.

The other comment was a suggestion to split the `CMD` variable assignment into multiple lines for better readability.

I tested the scripts by running multinode HPL with two DGX-1V 16G nodes using singularity (I fixed the typo when testing). Worked great.

File: workloads/bit/hpl/launch_hpl_experiment.sh [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+usegres=1
+hplai=0
+maxnodes=9999
+grestype=""
+mpiopts=""
+walltime=1:00:00
+verbose=0
+nores=0
+cruntime=singularity
+container="nvcr.io/nvidia/hpc-benchmarks:20.10-hpl"
```
Comment (avolkov1) [inline workloads/bit/hpl/launch_hpl_experiment.sh:40]: This needs to be updated to latest HPL container: "nvcr.io/nvidia/hpc-benchmarks:21.4-hpl"

**Review by avolkov1 — CHANGES_REQUESTED** [review]

**Review by avolkov1 — APPROVED** [review]
Great! Approved.

**Review by ajdecon — APPROVED** [review]
Merging based on @avolkov1 approval

**Pull request merged**